petabvis/plot_class.py

```diff
@@ -238,6 +238,13 @@ def add_warning(self, message: str):
         if message not in self.warnings:
             self.warnings = self.warnings + message + "\n"
 
+    def set_correlation_point_size(self, size):
+        """
+        Set the size of the points in the correlation_plot.
+        """
+        for scatter_plot in self.correlation_plot.listDataItems():
+            scatter_plot.setSize(size)
```

Comment:
Could add the units of `size` to the docstring in a `Parameters` section (and a typehint to the signature)